index.html

```diff
@@ -1275,7 +1275,71 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
       </ol>
     </section>
   </section>
-  <section> <h3>Using {{InteractionInput}} and {{InteractionOutput}}</h3>
+  
+  <section data-dfn-for="ActionInteractionOutput">
+    <h2>The <dfn>ActionInteractionOutput</dfn> interface</h2>
+    <p>
+      Belongs to the <a>WoT Consumer</a> conformance class.
+      An {{ActionInteractionOutput}} object is always created by the implementations
+      and exposes the data returned from <a>WoT Interactions</a> to application
+      scripts.
+    </p>
+    <p>
+      This interface exposes an action status object. Its implementation
+      will allow cancelling asynchronous actions and report the status of a long running action.
+    </p>
+    <pre class="idl">
+      enum ActionStatus {
+        /* "pending", Profile has pending. Not sure what it would mean? */
```

Comment:
I don't really see the reason for having it. Do you?

Reply:
No, since the Promise should fail if the operation could not be started.

Reply:
Thanks for the feedback. If I don't get any push-back the next days I will remove the option (it was added as a comment only anyway)

Reply:
done in https://github.com/w3c/wot-scripting-api/pull/561/commits/bb69456d1a11b2d3b3c26540d75cda9867fca39b